.travis.yml

```diff
@@ -2,6 +2,12 @@ language: generic
 sudo: required
 dist: trusty
 
+matrix:
+  include:
+    - env: CGAL_VER=4.8
+    - env: CGAL_VER=4.9
+    - env: CGAL_VER=4.10
+
```

Comment:
you can use env: matrix: instead of matrix: env -env -env... but I don't think it matters